package.json

```diff
@@ -87,7 +87,7 @@
     "eslint": "^7.0.0 || ^8.0.0 || ^9.0.0"
```

Comment:
Should we align here as well? Supporting all the way back to ESLint 7 but only Node 18 and newer feels a bit odd

Reply:
Yeah, this is what I did in the original PR (#471)

https://github.com/eslint-community/eslint-plugin-promise/commit/4db7f5d8379aa3a04cff72145f7681e97ccdaf3d

But I think that making that a separate PR for the change log was a good idea 😀